authentik/sources/ldap/models.py

```diff
@@ -99,6 +99,11 @@ class LDAPSource(Source):
         help_text=_("Property mappings used for group creation/updating."),
     )
 
+    password_login_update_internal_password = models.BooleanField(
```

Comment:
Maybe we need a better name for this

Reply:
I called this `update_local_user_password_after_bind` in my local branch^^

Reply:
Yeah I'm trying to use a name that will work for both LDAP and kerberos

Reply:
Sync seems to be the wrong word here as this is not done regularly but only on successful binds. If one is happy to use different terms for this mechanism in LDAP and kerberos I think that something like `update_{local,internal}_{user,password}_after_{bind,login}` is probably most clear as to what it does.

Reply:
we could go with `update_internal_password_after_login` indeed